### PR TITLE
Namespace update required for export, example command for easy refernace

### PR DESCRIPTION
**Pull request by iamraj007**
Export of ArgoCD backup need to happen on same namespace where argocd is running, a sample command will help save time and more PR for bug.

Signed-off-by: iamraj007 <47947075+iamraj007@users.noreply.github.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Review by blakepettersson — APPROVED** [review]

File: docs/operator-manual/disaster_recovery.md [hunk 1]
```diff
@@ -26,3 +26,5 @@ docker run -i -v ~/.kube:/home/argocd/.kube --rm argoproj/argocd:$VERSION argocd
 
 !!! note
     If you are running Argo CD on a namespace different than default remember to pass the namespace parameter (-n <namespace>). 'argocd admin export' will not fail if you run it in the wrong namespace.
+    example: kubectl config set-context --current --namespace=argocd
```
Comment (crenshaw-dev) [inline docs/operator-manual/disaster_recovery.md:29]: This is an example of how to avoid using the `-n` parameter. Can you rephrase to show that this is an _alternative_ to the `-n` parameter?

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]
